docs/template.md

```diff
@@ -16,28 +16,124 @@ Each template file must be set up in the following format:
 
+### Class Template Strings
+
+- `CLASS_TYPE`: ROBOT creates a class for each row of data with a that has a `TYPE` of `class` or `owl:Class`. The class type can be ONLY ONE OF:
+    - `subclass`: the created class will be asserted to be a subclass of each templated class expression (default)
+    - `disjoint`: the created class will be disjoint from each templated class expression, meaning the classes cannot share subclasses
+    - `equivalent`: the created class will be asserted to be equivalent to the intersection of all the templated class expressions
+- `C` **class expression**: If the template string starts with a `C` and a space then it will be interpreted as a class expression. The value of the current cell will be substituted into the template, replacing all occurrences of the percent `%` character. Then the result will be parsed into an OWL class expression. 
+    - ROBOT uses the same syntax for class expressions as Protégé: [Manchester Syntax](http://www.w3.org/2007/OWL/wiki/ManchesterSyntax). This means that an entity can be referred to by its `rdfs:label` (enclosing in single quotes if it has a space in it). 
+    - If it does not recognize a label, ROBOT will assume that you're trying to refer to a class by its IRI (or compact IRI). This can lead to unexpected behaviour, but it allows you to refer to classes (by IRI) without loading them into the input ontology. This is particularly useful when the input ontology would be too large, such as the NCBI Taxonomy.
+    - Properties in class expression **must** be referred to by label in order to be parsed.
+
+#### Example of Class Template Strings
+
+| Label | Entity Type | Class Type | Related classes | Other axioms |
+| --- | --- | --- | --- | --- |
+| LABEL | TYPE | CLASS_TYPE | C % | C part_of some % |
+| Class 2 | class | | Class 1 | |
+| Class 3 | class | disjoint | Class 2 | |
+| Class 4 | class | equivalent | | Class 3 |
+
+Class 2 will be a subclass of Class 1, as there is no included `CLASS_TYPE`. Class 3 will be disjoint with Class 2, and Class 4 will be equivalent to `part_of some 'Class 3'`.
+
+Manchester expressions can also be used within the cells, as long as they are enclosed in parentheses:
+
+| Label | Parent |
+| --- | --- |
+| LABEL | C % |
+| Class 4 | |
+| Class 5 | (part_of some 'Class 4') |
+
+In this template, Class 5 would be a subclass of `part_of some 'Class 4'`.
+
+### Property Template Strings
+
+- `PROPERTY_TYPE`: ROBOT creates a property for each row of data that has a `TYPE` of either an object or data property. The property type can be (any type followed by a \* can ONLY be used for object properties):
+    - **logical types**: these types link the created property to other properties (annotation properties can *only* use `subproperty`). You may only include ONE of these in each `PROPERTY_TYPE`.
```

Comment:
Does this column type allow splits?

Reply:
No - it should only be one of the logical types (e.g., not both equivalent and subproperty). I added a note to make this clear.